sys/kern/src/task.rs

```diff
@@ -809,6 +810,16 @@ pub fn priority_scan(
     choice.map(|(idx, _)| idx)
 }
 
+fn notify_supervisor(tasks: &mut [Task]) -> NextTask {
+    let supervisor_awoken =
+        tasks[0].post(NotificationSet(HUBRIS_TASK_STATE_CHANGE_NOTIFICATION));
```

Comment:
Yea, I saw that it was just moved, so I don't think we need to change it.  Was more of a curiosity.

Reply:
[Hubris reference documentation](https://hubris.oxide.computer/reference/#_what_is_the_supervisor) says that "The kernel can spot the supervisor because the supervisor always has task index 0, and is listed first in the app.toml"